articles/topics/index.md

```diff
@@ -0,0 +1,5 @@
+# Topic guides
+
+Come and explore some of the vast topics surrounding Intent Architect that would give you a much broader understanding of what it does and how it impacts Software Development as a whole.
+
+Please browse the panel to the left to locate the topic you are looking for.
```

Comment:
```suggestion
This is a collection of topic articles on Intent Architect which explain some of the rationale of Intent Architect and how it can vastly improve your software development practices.

Please refer to the side panel for the available articles.
```

The second sentence could be omitted if it's possible to embed the TOC somehow.